grpc/weaviate.proto

```diff
@@ -88,7 +92,7 @@ message TextArray{
   repeated string vals = 1;
 }
 message IntArray{
```

Comment:
would be great to change `IntArray{` to `IntArray {` here and in all following messages